packages/manager/apps/pci-ai-endpoints/src/hooks/token/useUserInfos.tsx

```diff
@@ -1,21 +1,29 @@
-import { useContext } from 'react';
-import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useQuery } from '@tanstack/react-query';
+import { getUserPermissions } from '@/data/api/policies/user.api';
 
 interface UseUserInfos {
-  isAdmin: boolean;
+  isAuthorized: boolean;
+  isLoading: boolean;
 }
 
-const USER_ADMIN_ROLE = 'ADMIN';
-
 const useUserInfos = (): UseUserInfos => {
-  const shellContext = useContext(ShellContext);
-  const user = shellContext.environment.getUser();
+  const { projectId } = useParams<{ projectId: string }>();
```

Comment:
It's not better take projectId as params directly and leave responsability to use useParams in page ?